mozilla-lockwise/lockwise-android/app/src/main/res/values/strings.xml

```diff
@@ -220,11 +220,11 @@
     <!-- This is the button to dismiss the final onboarding screen -->
     <string name="finish">Finish</string>
     <!-- This description of the app functionality is shown on the final onboarding screen -->
-    <string name="access_description">Access logins saved to Firefox from your phone</string>
+    <string name="onboarding_access_description">Access logins saved to Firefox from your phone</string>
     <!-- This description of the app security is shown on the final onboarding screen. %s will be replaced with the security link. -->
```

Comment:
@eliserichards I merged this to unblock the situation and expose strings in Pontoon. However, could you please update the comment here in a following commit? Unless I'm confused, it seems to still call out the placeholder. Thanks!